docs/source/x_ext.rst

```diff
@@ -65,7 +65,7 @@ The CV-X-IF specification contains the following parameters:
   |                              |                        |               | to MXLEN.                                                          |
   |                              |                        |               | The |processor| determines the legal values for this parameter.    |
   +------------------------------+------------------------+---------------+--------------------------------------------------------------------+
-  | ``X_MISA``                   | logic [31:0]           | 32'b0         | MISA extensions implemented on the eXtension interface.            |
+  | ``X_MISA``                   | logic [25:0]           | 32'b0         | MISA extensions implemented on the eXtension interface.            |
   |                              |                        |               | The |processor| determines the legal values for this parameter.    |
```

Comment:
I need to check for @Silabs-ArjanB 's input here. It could be that the restriction for legal values came from the fact that the parameter was defined as 31:0. In that case, some bits had to be 0. If we limit to 25:0, there might still be some values that cannot be legal, e.g. a copro setting I=1.

Reply:
Only the processor uses this parameter. The processor defines what values are acceptable. See for example https://docs.openhwgroup.org/projects/cv32e40x-user-manual/en/latest/integration.html and specifically https://docs.openhwgroup.org/projects/cv32e40x-user-manual/en/latest/control_status_registers.html#csr-misa

Reply:
resolved in 6a0209c